wechat/controllers/wechat_controllers.py

```diff
@@ -32,3 +32,63 @@ def micropay(self):
             return """<xml><return_code><![CDATA[SUCCESS]]></return_code><return_msg><![CDATA[OK]]></return_msg></xml>"""
         else:
             return """<xml><return_code><![CDATA[FAIL]]></return_code><return_msg><![CDATA[Signature failure]]></return_msg></xml>"""
+
+    @http.route('/wechat/miniprogram/authenticate', type='json', auth='none')
```

Comment:
when you use ``auth='none'``, ``self.env`` is not available, because database is unknow. You can use ``public`` here